build.sbt

```diff
@@ -36,3 +36,19 @@ lazy val zioNio = project
       "org.scalaz"     %% "testz-specs2"     % "0.0.5" % Test
     )
   )
+
+lazy val docs = project
+  .in(file("zio-nio-docs"))
+  .settings(
+    skip.in(publish) := true,
+    moduleName := "zio-nio-docs",
+    scalacOptions -= "-Yno-imports",
+    scalacOptions -= "-Xfatal-warnings",
+    libraryDependencies ++= Seq(
+      "dev.zio"      %% "zio"        % "1.0.0-RC11-1",
+      "dev.zio"      %% "zio-actors" % "0.0.1+10-51dde802",
```

Comment:
As said in the comment, `zio-actors` were pointed for the sake of documentation, no need to implement remoting here.

Reply:
Sure, removed